README.md

```diff
@@ -1,6 +1,42 @@
-# Keycloak-js
+# Keycloak JS
 
-Client-side JavaScript library that can be used to secure web applications and integrate them with Keycloak.
+Keycloak is an Open Source Identity and Access Management solution for modern Applications and Services.
 
-TODO
+This reporsitory contains the client-side JavaScript library that can be used to secure web applications and integrate them with Keycloak.
 
+## Help and Documentation
+
+* [Documentation](https://www.keycloak.org/documentation.html)
+* [Keycloak JavaScript adapter guide](https://www.keycloak.org/securing-apps/javascript-adapter) - The specific guide for this client
+* [User Mailing List](https://groups.google.com/g/keycloak-user) - Mailing list for help and general questions about Keycloak
+* [Issue Tracker](https://github.com/keycloak/keycloak-nodejs-connect/issues) - Issue tracker for bugs and feature requests
+
+## Reporting Security Vulnerabilities
+
+If you've found a security vulnerability, please look at the [instructions on how to properly report it](http://www.keycloak.org/security.html)
+
+## Reporting an issue
+
+If you believe you have discovered a defect in the keycloak-js adapter please open an issue in our [Issue Tracker](https://github.com/keycloak/keycloak-js/issues).
```

Comment:
If we are linking the issue tracker above, do we need to repeat it here (or the other way around)?

Reply:
We can remove the previous link if you want. I think this section is OK and I would maintain it.

Reply:
Yep, I have removed the previous link and commented here that the tracker can also be used for feature/enhancement requests.